integration/go.mod

```diff
@@ -5,6 +5,7 @@ go 1.18
 require github.com/kr/pretty v0.3.0
 
 require (
+	github.com/kinvolk/inspektor-gadget v0.8.0 // indirect
```

Comment:
TODO: How to use the events of the commit we are testing?

Reply:
I know we can now use [go workspaces](https://go.dev/doc/tutorial/workspaces) for situations like these but if we decide to get rid of integration test image via #944 we don't really need this module. Right?

Reply:
Yep. that's also my understanding. If we are not missing something, that's another good reason to get rid of that image.

Reply:
I implemented a solution using go workspaces, but it could still be improved once #944 is solved. See https://github.com/kinvolk/inspektor-gadget/pull/945#discussion_r974174068.